build.yaml

```diff
@@ -243,25 +251,26 @@ common_cli:
     # We have a devpi cache in AWS which we will consume instead of going
     # upstream to the PyPi servers.
     # We specify that devpi caching server using -i \$PIP_INDEX_URL
-    # which requires as to include --trusted_host as we are not (yet) using
+    # which requires as to include --trusted-host as we are not (yet) using
     # SSL on our caching box.
     # The --trusted-host option is only available with pip 7.
     #
+    if [ \${PIP_INDEX_URL} ]; then
+        PIP_ADDITIONAL_OPTIONS="\${PIP_ADDITIONAL_OPTIONS} -i \${PIP_INDEX_URL} "
```

Comment:
it may not be available, the contents of those vars are loaded to the slave. But not all slaves have them (OSX builds on cloudbees don't have them).
So it's better to check if they're defined (this was actually Tom's suggestion a cpl of weeks ago)